packages/abtest/src/Abtest.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * A class that interacts with WP.com A/B tests.
+ *
+ * @package automattic/jetpack-abtest
+ */
+
+namespace Automattic\Jetpack;
+
+use Automattic\Jetpack\Connection\Client;
+use Automattic\Jetpack\Error;
+
+/**
+ * This class provides an interface to the WP.com A/B tests.
+ */
+class Abtest {
+	/**
+	 * A variable to hold the tests we fetched, and their variations for the current user.
+	 *
+	 * @access private
+	 *
+	 * @var array
+	 */
+	private $tests = array();
+
+	/**
+	 * Retrieve the test variation for a provided A/B test.
+	 *
+	 * @access public
+	 *
+	 * @param string $test_name Name of the A/B test.
+	 * @return mixed|null A/B test variation, or null on failure.
+	 */
+	public function get_variation( $test_name ) {
+		$variation = $this->fetch_variation( $test_name );
+
+		// If there was an error retrieving a variation, conceal the error for the consumer.
+		if ( is_wp_error( $variation ) ) {
+			return null;
+		}
+
+		return $variation;
+	}
+
+	/**
+	 * Fetch and cache the test variation for a provided A/B test from WP.com.
+	 *
+	 * @access protected
+	 *
+	 * @param string $test_name Name of the A/B test.
+	 * @return mixed|Automattic\Jetpack\Error A/B test variation, or Automattic\Jetpack\Error on failure.
```

Comment:
> Automattic\Jetpack\Error A/B test variation

Is this a correct return type?

Reply:
😉 Yeah, I understand how the mix of slashes could be confusing :) 

It's a valid type - `Automattic\Jetpack\Error` is the type, and the rest is the description.